packages/design-system/src/components/N8nButton/Button.vue

```diff
@@ -68,11 +66,20 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		outline: {
+			type: Boolean,
+			default: false,
+		},
+		text: {
+			type: Boolean,
+			default: false,
+		},
 		icon: {
+			type: String,
 		},
 		round: {
 			type: Boolean,
-			default: true,
+			default: false,
 		},
 		circle: {
```

Comment:
maybe we can delete these two... based on a quick search, we are only using `circle` in `icon button` and that needs to be changed... also `round` does not seem to be used anymore

Reply:
Good point. Done! :)